docs/_book.yaml

```
@@ -34,6 +34,8 @@ upper_tabs:
       - heading: "Tutorials"
       - title: "Get started with qsimcirq"
         path: /qsim/tutorials/qsimcirq
+      - title: "Quantum simulation on GCP with _Cirq_ and _qsim"
```

```suggestion
      - title: "Quantum simulation on GCP with Cirq and qsim"
```
No styles in leftnav